ground-core/conf/config.yml

```diff
@@ -1,10 +1,10 @@
 dbType: neo4j
 dbName: ground
-dbHost: localhost
+dbHost: neo
```

Comment:
those are the hostnames of the docker containers. I could just add this file to my gitignore so that my changes don't affect my PRs